build/js/placeholders.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+function assignPlaceholders() {
+    // swaps html placeholders with custom ones
+    var formList = {
+        first_name: 'First name',
+        last_name: 'Last name',
+        textarea_1: 'Text1',
+        textarea_2: 'Text2',
+        email: 'email@gmail.com',
+        password: 'Password',
+        vid_number: 'VID',
+        tickets_count: '1'
+    };
+    for (var placeholderText in formList) {
```

Comment:
When using this kind of loop on an object, you should always add this kind of check :
```javascript
if (!formList.hasOwnProperty(placeholderText)) continue;
```
as the enumerable properties defined by Object's prototype will be used in iteration.